homeassistant/components/swiss_public_transport/__init__.py

```diff
@@ -109,6 +114,10 @@ async def async_migrate_entry(
             config_entry, unique_id=new_unique_id, minor_version=2
         )
 
+    if config_entry.version < 2 and config_entry.minor_version < 3:
```

Comment:
```suggestion
    if config_entry.version < 2:
```
Enough to check the major version here